### PR TITLE
Adding May update (and a small fix in April update)

### PR DESCRIPTION
**Pull request by sublibra**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Deploy preview for *core-website* ready!

Built with commit f8b9ab3771dacb5eda066c84f908712ba4bbf70b

https://deploy-preview-510--core-website.netlify.com

File: docs/updates/2019-05.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+## Release of corectl 1.0.0
+
+After a final touch up we released corectl as a stable 1.0.0. As described in
+[previous update](2019-04.md#corectl-100-beta-release) we changed the command structure and with this release we
+confirmed it.
+
+### Installation through snap
+
+For our Linux users out there we have added support for installation of latest corectl through
+[snap](https://snapcraft.io/). Install with:
```
Comment (wennmo) [inline docs/updates/2019-05.md]: We should also mention the auto-alias `corectl` for the binary

File: docs/updates/2019-05.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+### Other fixes
+
+* Fix bash completion in Widows Subsystem for Linux (WSL)
+* Additional information in the version command
+
+## New file streaming API in engine
+
+A new file streaming API has been released in engine, giving us a new way of working with remote files in engine.
```
Comment (wennmo) [inline docs/updates/2019-05.md]: Skip the last "in engine"

File: docs/updates/2019-05.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+Functionality wise, the feature is similar to the folder connector used to access local files. If you are familiar with
+this you will feel home since load script functions such as `Load`, `Store` and `FileSize()` will work as expected.
+
+The feature is intended to enable a user to load or store files from external sources such as an S3 bucket, Dropbox or
+other remote storage services. With the release we have provided a
+[S3 file connector](https://github.com/qlik-oss/core-grpc-s3-file-connector) and a
```
Comment (wennmo) [inline docs/updates/2019-05.md]: Highlight that it is an example connector

File: docs/updates/2019-05.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+[tutorial](../tutorials/data-loading/remote-files.md) explaining its usage.
+
+## Enhancements in catwalk
+
+We have continued to work with the feedback we receive (keep it coming!) and have some update you might like.
```
Comment (wennmo) [inline docs/updates/2019-05.md]: plural updates

File: docs/updates/2019-05.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+### Other fixes
+
+* Fix bash completion in Widows Subsystem for Linux (WSL)
+* Additional information in the version command
+
+## New file streaming API in engine
+
+A new file streaming API has been released in engine, giving us a new way of working with remote files.
```
Comment (wennmo) [inline docs/updates/2019-05.md]: I think we should be consistent with the naming of this API. In the documentation we refer to it by `File Connector API`. Maybe also attach a link to the name pointing to the API introduction i.e. https://core.qlik.com/services/qix-engine/apis/file-loading/introduction/

File: docs/updates/2019-05.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+A common activity during data modeling is sanitation of data. Dirty data such as numbers with trailing spaces, null
+values or incorrect decimal character may introduce errors in measurements which need to be found and fixed. To support
+these cases numeric values will be right aligned to differentiate against string values
+
+![screenshot](../images/catwalk-numeric-right.png)
+
+### Use of hamus.js
+
+Since the release of hamus.js we could refactor some of the catwalk code to use the hamus.js hooks for engine
```
Comment (hrigner) [inline docs/updates/2019-05.md]: add link to npm https://www.npmjs.com/package/hamus.js or github https://github.com/qlik-oss/hamus.js

**Pull request merged**